src/game_player.cpp

```diff
@@ -215,7 +215,23 @@ void Game_Player::UpdateScroll(int amount, bool was_jumping) {
 		return;
 	}
 
-	Game_Map::Scroll(sx * amount, sy * amount);
+	int move_sx = 0;
```

Comment:
It looks like something was messed up here. github says this is a merge commit. Maybe rebase once more?